src/views/docs/en/tutorials/beyond-hello-world.md

```diff
@@ -0,0 +1,485 @@
+---
+title: 'Going beyond "Hello World"'
+category: Tutorials
+description: Next steps in developing an application with Architect
+---
+
+Let's go beyond a simple "Hello world" HTTP function by incrementally adding features to a new Architect application. For this tutorial we'll:
+
+* Add "pragmas" to our `./app.arc` manifest file
+* Use features of the `arc` CLI to generate scaffolding and code
+* Preview our application locally with Sandbox
+* Use "vanilla" JavaScript and HTML
+* Forego a front end framework
+
+## Get started
+
+> If you haven't already, check the Quickstart guide to get set up.
+
+As an example, we'll use a simple meal delivery service as a model. Starting from what is created by running:
+
+```sh
+npm @architect init dinner-delivery
+```
+
+The beginning of _Dinner Delivery™️_ lives in our new `app.arc`
+
+```arc
+# ./app.arc
+@app
+dinner-delivery
+
+@http
+get /
+```
+
+## `@http` routes
+
+Next, we can extend the number of HTTP routes by filling in some obvious endpoints for a meal delivery service (we'll skip user account management and authentication for this tutorial) replace the `@http` block with:
+
+```arc
+@http
+get  /
+get  /menu # see today's dinner offerings
+post /orders # create, update, delete an order
+get  /admin # see today's orders
+```
+
+### Generate scaffolding
+
+After updating our `app.arc`, we can have the Architect CLI help with scaffolding. From the root of our project:
+
+```sh
+npx arc init
+```
+
+We'll get a new folder structure like:
+
+```sh
+.
+├── src
+│   └── http 
+│       ├── get-admin
+│       ├── get-index
+│       ├── get-menu
+│       └── post-orders
+└── app.arc
+```
+
+> At any step in this tutorial start up the local Sandbox with `npx arc sandbox` and navigate to `localhost:3333`.
+
+## `@static` assets
+
+We know we'll want to serve things like images and CSS that aren't dynamically created by our functions. To enable a static asset server (backed by S3), add a single pragma to the project's manifest:
+
+```arc
+@static
+```
+
+Now, by default, files in our project's `./public/` directory can be accessed via HTTP at `/_static/`.
+
+This example will reference `/_static/styles.css`. It provides some simple layout and styles; I won't include the contents here.
+
+## `@shared` code
+
+Architect supports sharing code across all functions. Perfect for utilities and common functionality. Sandbox will help copy our shared code while running and `arc deploy` will hydrate any shared code to each created Lambda.
```

Comment:
I think we can expand upon this a little bit. Maybe some more verbiage on how when running `sandbox` any changes to code in the shared folder will automatically be copied to all lambda's while `deploy` handles coping all the shared code during the deploy to prod step.